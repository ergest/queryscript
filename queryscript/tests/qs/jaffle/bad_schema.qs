type raw_order {
    bool_value bool,
}

export let orders [raw_order] = load('data/raw_orders.csv');

-- This should throw a runtime error
SELECT MAX(bool_value) FROM orders;
