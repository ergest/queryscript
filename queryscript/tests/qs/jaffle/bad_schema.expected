{
    "compile_errors": [],
    "decls": {
        "let orders": [{
        	bool_value Boolean,
        }],
        "type raw_order": {
        	bool_value Boolean,
        },
    },
    "queries": [
        Err(
            ArrowError {
                source: ParseError(
                    "Error while parsing value 1 for column 0 at line 1",
                ),
                backtrace: None,
            },
        ),
    ],
}